src/components/Stats/index.js

```diff
@@ -0,0 +1,206 @@
+import React from 'react';
+import { withProps } from 'recompose';
+import filesize from 'filesize';
+import { get } from 'lodash';
+import Stats from './Stats';
+
+export const FileRepoStats = withProps(() => ({
```

Comment:
This component.. its both the stats query and the presentation right? can we decouple that? we need to get those agg stats into dynamo db as well

Reply:
Sure, what are you thinking? `Stats` is the presentation + data fetching. But the query definitely is just a config thats passed to it. So we could easily move those to a separate file like `predefinedQueries` or something.